docs/howto/python.md

```diff
@@ -0,0 +1,131 @@
+specified, the buildpack will provide the default version, which can be seen in
+the `buildpack.toml` file.
+
+## <a id="package-management-options"></a> Package Management Options
+
+With the Python CNB, there are three options available for package management
+depending on your application:
+
+* Using [Pip](https://pip.pypa.io)
+* Using [Pipenv](https://pypi.org/project/pipenv)
+* Using [Miniconda](https://docs.conda.io/en/latest/miniconda.html)
+
+You can find specific information for each option below.
+
+### <a id="package-management-with-pip"></a> Package Management with Pip
+
+Pip is a popular option for managing third-party application dependencies for
+Python apps.  Including a valid `requirements.txt` file at the root of your app
+source code triggers the pip installation process by the buildpack. The
+buildpack will install the application packages and make it available to the
+app.
+
+The buidpack allows you to configure the version of Pip to be used in the
+installation process. You can set this using the `$BP_PIP_VERSION` variable
+during build. When specifying a version of Pip, you must choose a version that
+is available within the buildpack. The supported versions can be found in the
+[release notes](https://github.com/paketo-community/python/releases/latest).
+
+### <a id="package-management-with-pipenv"></a> Package Management with Pipenv
+
+Pipenv is another common option for managing dependencies. Including a valid
+`Pipfile` file at the root of your app source code triggers the pipenv
+installation process by the buildpack. The buildpack will install the
+application packages and make it available to the app.
+
+The buidpack allows you to configure the version of Pipenv to be used in the
+installation process. You can set this using the `$BP_PIPENV_VERSION` variable
+during build. When specifying a version of Pipenv, you must choose a version
+that is available within the buildpack. The supported versions can be found in the
+[release notes](https://github.com/paketo-community/python/releases/latest).
+
+The buildpack also takes into consideration the Python version requirement
+specified by `Pipfile.lock`, but `BP_CPYTHON_VERSION` takes precedence over
+this as discussed in [this section above](#specifying-python-version).
+
+### <a id="using-miniconda"></a> Using Miniconda
+
+Miniconda is a package management and environment management system supported
+by the Python buildpack. The builpack will create or update a conda environment
+from an `environment.yml` file or a `package-list.txt` file located at the root
+of the app source code.
+
+Configuring a version of miniconda is not supported.
+
+## Install a Custom CA Certificate
+Python Buildpack users can provide their own CA certificates and have them
+included in the container root truststore at build-time and runtime by
+following the instructions outlined in the [CA
+Certificates](docs/reference/configuration/#ca-certificates)
```

Comment:
This link is broken (missing "https://paketo.io"?)

Reply:
Yes but it should be fixed when this dir is moved to the paketo-website repo.
See #281 docs are created here such that it can be easily copy-pasted into there. That block is taken from https://raw.githubusercontent.com/paketo-buildpacks/paketo-website/main/content/docs/howto/nodejs.md